terraform/modules/weblate/ecs.tf

```diff
@@ -29,38 +29,14 @@ module "app" {
-
-    weblate_admin_email                    = ""
-    weblate_github_username                = ""
-    weblate_github_token                   = ""
-    weblate_github_host                    = ""
-    weblate_gpg_identity                   = ""
-    weblate_social_auth_github_key         = ""
-    weblate_social_auth_github_secret      = ""
-    weblate_social_auth_github_org_key     = ""
-    weblate_social_auth_github_org_secret  = ""
-    weblate_social_auth_github_org_name    = ""
-    weblate_social_auth_github_team_key    = ""
-    weblate_social_auth_github_team_secret = ""
-    weblate_social_auth_github_team_id     = ""
-
-    # weblate_admin_email = data.aws_secretsmanager_secret.shared_secrets["weblate_admin_email"].arn
-    # weblate_github_username = data.aws_secretsmanager_secret.shared_secrets["weblate_github_username"].arn
-    # weblate_github_token = data.aws_secretsmanager_secret.shared_secrets["weblate_github_token"].arn
-    # weblate_github_host = data.aws_secretsmanager_secret.shared_secrets["weblate_github_host"].arn
-    # weblate_gpg_identity = data.aws_secretsmanager_secret.shared_secrets["weblate_gpg_identity"].arn
-    # weblate_social_auth_github_key = data.aws_secretsmanager_secret.shared_secrets["weblate_social_auth_github_key"].arn
-    # weblate_social_auth_github_secret = data.aws_secretsmanager_secret.shared_secrets["weblate_social_auth_github_secret"].arn
-    # weblate_social_auth_github_org_key = data.aws_secretsmanager_secret.shared_secrets["weblate_social_auth_github_org_key"].arn
-    # weblate_social_auth_github_org_secret = data.aws_secretsmanager_secret.shared_secrets["weblate_social_auth_github_org_secret"].arn
-    # weblate_social_auth_github_org_name = data.aws_secretsmanager_secret.shared_secrets["weblate_social_auth_github_org_name"].arn
-    # weblate_social_auth_github_team_key = data.aws_secretsmanager_secret.shared_secrets["weblate_social_auth_github_team_key"].arn
-    # weblate_social_auth_github_team_secret = data.aws_secretsmanager_secret.shared_secrets["weblate_social_auth_github_team_secret"].arn
-    # weblate_social_auth_github_team_id     = data.aws_secretsmanager_secret.shared_secrets["weblate_social_auth_github_team_id"].arn
+    weblate_admin_password         = aws_secretsmanager_secret.app_secrets["weblate_admin_password"].arn
+    weblate_admin_email            = data.aws_secretsmanager_secret.shared_secrets["weblate_admin_email"].arn
+    postgres_password              = aws_secretsmanager_secret.app_secrets["postgres_password"].arn
+    redis_password                 = aws_secretsmanager_secret.app_secrets["redis_password"].arn
+    weblate_email_host_password    = aws_secretsmanager_secret.app_secrets["weblate_email_host_password"].arn
+    weblate_github_username        = data.aws_secretsmanager_secret.shared_secrets["weblate_gpg_identity"].arn
+    weblate_github_username        = data.aws_secretsmanager_secret.shared_secrets["weblate_github"].arn
```

Comment:
Is this value supposed to be immediately overwritten here?

Reply:
well spotted, thanks. I've fixed that